mteb/abstasks/AbsTaskBitextMining.py

```diff
@@ -130,6 +130,13 @@ def _evaluate_subset(
             pair_columns=pairs,  # type: ignore
             **kwargs,
         )
+        sentences = [row[col] for col in pairs[0] for row in data_split]
+
+        model.load_task_sample(
+            sentences,
+            task_name=self.metadata.name,
+            **encode_kwargs,
+        )
```

Comment:
Won't this break with other models?

Reply:
Couldn't this be moved to the model's encode function? No need to make changes in the abstasks